website/package.json

```diff
@@ -25,16 +26,17 @@
 		"firebase-admin": "^11.3.0",
 		"firebase-functions": "^4.1.0",
 		"next": "13.0.4",
+		"next-transpile-modules": "^10.0.0",
 		"react": "18.2.0",
 		"react-dom": "18.2.0",
 		"typescript": "4.9.3"
 	},
 	"devDependencies": {
 		"@testing-library/jest-dom": "^5.16.5",
 		"@testing-library/react": "^13.4.0",
+		"isomorphic-fetch": "^3.0.0",
```

Comment:
> Is this one still required considering Node 18 is now the LTS and supports `fetch`?

Good question. Worth trying to update to 18 👍 

> 
> If so, nitpick, should this be under `dependencies` instead of `devDependencies`?

Done.